server/lib/messages/phone-numbers-to-send-to.js

```diff
@@ -27,7 +27,8 @@ function isMessageForSingleOfficeAndDoesPhoneNumberSubscribeToAllOfficesInTheAre
 module.exports = (users, message) => {
   const phoneNumbers = []
   users
-    .filter(x => x.active && message.orgCodes.includes(x.orgCode))
+    .filter(x => x.active)
```

Comment:
Keep each filter simple.
I considered using a named function for the check on `allOrgs` but thought it read simple enough as is.